app/tasks/doctor.php

```diff
@@ -9,11 +9,13 @@
 use Utopia\App;
 use Utopia\CLI\Console;
 use Utopia\Domains\Domain;
+use Utopia\Validator\Text;
 
 $cli
     ->task('doctor')
     ->desc('Validate server health')
-    ->action(function () use ($register) {
+    ->param('interactive', 'n', new Text(1), 'Run an interactive session', true)
```

Comment:
Good approach to make sure existing functionality still works!

Would you please also add a param to allow setting the address so that you can still do non-interactive but set a different email?

Reply:
Hey there, this is done and tested. I've also linted the file. Take a look when you can, thanks.